unstructured/ingest/v2/cli/cmds/elasticsearch.py

```diff
@@ -104,9 +104,56 @@ def get_cli_options() -> list[click.Option]:
         return options
 
 
+@dataclass
+class ElasticsearchCliUploadStagerConfig(CliConfig):
+    @staticmethod
+    def get_cli_options() -> list[click.Option]:
+        options = [
+            click.Option(
+                ["--index-name"],
```

Comment:
How does this work since we already have --index-name in line 92? I guess it will work because this is part of the destination connector. But what if we are going from ES to ES?

Reply:
It would still exist in two different parts of the CLI call, one would be before the destination connector, one would be after. So the dictionary coming back from click wouldn't conflict.